packages/openapi/src/decorators.ts

```diff
@@ -214,3 +218,52 @@ export function resolveInfo(program: Program, entity: Namespace): AdditionalInfo
 function omitUndefined<T extends Record<string, unknown>>(data: T): T {
   return Object.fromEntries(Object.entries(data).filter(([k, v]) => v !== undefined)) as any;
 }
+
+function validateObject(context: DecoratorContext, typespecType: TypeSpecValue) {
```

Comment:
```suggestion
function validateAdditionalInfoModel(context: DecoratorContext, typespecType: TypeSpecValue) {
```

Reply:
here otherwise you are just silently ignoring other cases without being explicit that they cannot happen